src/stirling/source_connectors/perf_profiler/perf_profile_connector.cc

```diff
@@ -61,6 +61,12 @@ PerfProfileConnector::PerfProfileConnector(std::string_view source_name)
       sampling_period_(
           std::chrono::milliseconds{1000 * FLAGS_stirling_profiler_table_update_period_seconds}),
       push_period_(sampling_period_ / 2),
+      profiler_state_overflow_gauge_(
+          BuildGauge("perf_profiler_overflow_gauge",
+                     "Fraction of the overflow, e.g. overflowed by 1.25x the expected amount.")),
```

Comment:
Shouldn't this value should be < 1? Since we are tracking `actual / expected` if there is overflow I would assume actual would be less than expected.

Reply:
The "error" condition is triggered when the actual count exceeds the threshold amount. The threshold amount is set 1/2 way between the "expected" value and the "over-provisioned" max value. Our current over-provision factor is 3x. That gives us: expected=1x, error-threshold=2x, configured-max=3x. So when we see this ratio reported, hopefully we see a number that is closer to 2x and not exceeding 3x. Will update the help message to be more precise.